docs/conf/helpop-full.conf.example

```diff
@@ -750,9 +751,11 @@ can be negated by any eline that matches.">
 
 Sends a message to all +w users.">
 
+# NOTE: Use '\\s' instead of '\s' (in the mask example below)
```

Comment:
Can't we just force `<config format="xml">` for this file and then let people who want to change that fix it themself?

Reply:
I didn't think that was per file. If it is then sure that would be acceptable.